src/main/scala/cognite/spark/v1/SequencesRelation.scala

```diff
@@ -0,0 +1,150 @@
+package cognite.spark.v1
+
+import java.time.Instant
+
+import cats.data.NonEmptyList
+import cats.effect.IO
+import cats.implicits._
+import cognite.spark.v1.SparkSchemaHelper.{asRow, fromRow, structType}
+import com.cognite.sdk.scala.common.{WithExternalId, WithId}
+import com.cognite.sdk.scala.v1.resources.SequencesResource
+import com.cognite.sdk.scala.v1._
+import fs2.Stream
+import io.scalaland.chimney.dsl._
+import org.apache.spark.sql.sources.{Filter, InsertableRelation}
+import org.apache.spark.sql.types.{DataTypes, StructType}
+import org.apache.spark.sql.{Row, SQLContext}
+
+class SequencesRelation(config: RelationConfig)(val sqlContext: SQLContext)
+    extends SdkV1Relation[SequenceReadSchema, Long](config, "sequences")
+    with InsertableRelation
+    with WritableRelation {
+  import CdpConnector._
+
+  override def getStreams(filters: Array[Filter])(
+      client: GenericClient[IO, Nothing],
+      limit: Option[Int],
+      numPartitions: Int): Seq[Stream[IO, SequenceReadSchema]] =
+    // TODO: partitions and filters!!!
```

Comment:
I don't think they support partitions?

Reply:
They don't, unfortunatelly :/ Hopefully there is not gonna be too many sequences in a single project